src/routes/board/withBoardProps.js

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import { compose } from "redux";
+
+import { handCheck } from "lib/handCheck";
+import {
+  check,
+  fold,
+  raise,
+  replace,
+  startGame,
+  advancePhase,
+  aiBet,
+  aiHand,
+  aiMoney,
+  deck,
+  phase,
+  playerBet,
+  playerHand,
+  playerMoney,
+  pot,
+} from "models/game";
+import { logMessages } from "models/log";
+import { withModelProps } from "aa-minimal-core-lib/components/model-props";
+
+const withBoardProps = (Component) => (props) => {
+  const {
+    playerHand,
+    deck,
+    phase,
+    playerMoney,
+    playerBet,
+    aiMoney,
+    aiBet,
+    pot,
+    logMessages,
+    fold,
+    check,
+    raise,
+  } = props;
+
+  const playerStats = {
+    money: playerMoney,
+    bid: playerBet,
+    strength: phase >= 1 ? handCheck(playerHand)[1] : null,
```

Comment:
Make phases more declarative and use reference data. For example

const PHASES = Object.freeze({
  NOT_STARTED: 0,
  ...
}); ..etc

Reply:
PHASES added